CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+cmake_minimum_required(VERSION 3.6)
+
+if (DEFINED ENV{VCPKG_ROOT})
+  set(CMAKE_TOOLCHAIN_FILE "$ENV{VCPKG_ROOT}/scripts/buildsystems/vcpkg.cmake")
+endif ()
+
+set(n clinfo)
+project(${n})
+
+find_package(OpenCL REQUIRED)
+
+add_executable(${n} src/${n}.c)
+target_include_directories(${n} PUBLIC src)
```

Comment:
This can be simplified to a single function call: 
```cmake
target_include_directories(${n} PRIVATE src ${OpenCL_INCLUDE_DIRS})
```

If we use an imported target (see below), this comment can obviously be ignored.